src/server/multi_command_squasher.cc

```diff
@@ -181,6 +181,9 @@ void MultiCommandSquasher::Run() {
 
   ExecuteSquashed();  // Flush leftover
 
+  // Set last txid.
+  cntx_->last_command_debug.clock = cntx_->transaction->txid();
+
```

Comment:
What if its non-atomic? We need to set it after every schedule-hop-conclude cycle

Reply:
it's used for testing. Tests can not observe this field in the middle of command execution 🤷🏼